py/specter/test/test_psf.py

```diff
@@ -486,23 +486,23 @@ def test_xyrange(self):
         self.assertGreater(xmax, 0)
 
 #- Test Pixellated PSF format
-class TestPixPSF(TestPSF):
+class TestPixPSF(GenericPSFTests,unittest.TestCase):
```

Comment:
The reason is that the test framework was trying to run the *base class* as a test.  But the base class is meaningless without the initializations in the derived classes.  In other words the base class is an abstract class that should never be instantiated.  The multiple inheritance allows the test framework to skip the base class and only test the derived classes.